README.md

```diff
@@ -10,6 +10,8 @@ The source version will increase over time as newer releases of VeriStand are re
 The following dependencies are used by some or all of the included components of this repository.
 
 ### VI Package Manager (hosted on the LabVIEW Tools Network)
+To quickly install all of the dependent packages, use the [VI Package](vi-packages/ni_lib_veristand_custom_device_dev_dependencies-1.0.0.4.vip) found in this repo. This is an empty package that depends on the files listed below, allowing you to install all packages for a specific version of LabVIEW with one installation.
```

Comment:
If the package ever gets rebuilt, we'll need to come back and update this one link, which we will almost certainly forget to do. This will mean the link is broken. Just link to the `vi-packages` directory and tell the user to install the `.vip` file that resides there.